hostinfo/cron.lua

```diff
@@ -15,48 +15,48 @@ limitations under the License.
 --]]
 local HostInfo = require('./base').HostInfo
 
-local los = require('los')
-local readCast = require('./misc').readCast
+local m = require('./misc')
+local getInfoByVendor = m.getInfoByVendor
+local readCast = m.readCast
 local async = require('async')
 local fs = require('fs')
 local path = require('path')
-local sigar = require('sigar')
 
---[[ Pluggable auth modules ]]--
+--[[ Cron ]]--
 local Info = HostInfo:extend()
-function Info:initialize()
-  HostInfo.initialize(self)
-end
-
 function Info:run(callback)
-  if los.type() ~= 'linux' then
-    self._error = 'Unsupported OS for pluggable auth module definitions'
-    return callback()
-  end
+  local errTable, options, dir, outTable, vendorInfo, deb, rhel
+  errTable, outTable = {}, {}
 
-  local cdir, vendor, errTable
-  vendor = sigar:new():sysinfo().vendor:lower()
-  errTable = {}
-  if vendor == 'ubuntu' or vendor == 'debian' then
-    cdir = '/var/spool/cron/crontabs'
-  elseif vendor == 'rhel' or vendor == 'centos' then
-    cdir = '/var/spool/cron'
-  else
-    self._error = 'Could not determine OS'
+  deb = {dir = '/var/spool/cron/crontabs' }
```

Comment:
local deb and local rhel and local options

try and keep the locals close to together.